jenkins/suites-groups.sh

```diff
@@ -132,11 +149,5 @@ check() {
   fi
   echo "************************"
 
-  exit ${failure}
+  return ${failure}
 }
-
-case "$1" in
```

Comment:
I would leave this part. It allows us to run the script locally just to check if everything is OK without the need of sourcing it to the current shell.

Reply:
I've never seen such type of a call:
```
${WORKSPACE}/suites-groups.sh check ${WORKSPACE}/mysql-test-run.pl
```
The traditional usage is to `source` functions from other scripts/includes and not to `source` variables like `WORKER_x_MTR_SUITES`.
I converted `suites-groups.sh` just to consist of 2 functions `get_suites()` and `check_suites()`. The `WORKER_x_MTR_SUITES` variables will be defined only if one of functions was called.